src/components/Editor.jsx

```diff
@@ -1,6 +1,6 @@
 import React, { PropTypes } from 'react';
 import CodeMirror from 'react-codemirror';
-import _ from 'lodash';
+import isEqual from 'lodash/isEqual';
```

Comment:
We're not sacrificing speed because it performs exactly the same as it did before. The only difference this PR introduces is the decreased bundle size.